controller/ToolModule.php

```diff
@@ -115,4 +117,19 @@ public function launch()
      * run() contains the actual tool's controller
      */
     abstract public function run();
+
+    /**
+     * Build launch data from request and add extra parameters
+     *
+     * @param common_http_Request $request
+     * @return LtiLaunchData
+     * @throws ResolverException
+     */
+    protected function buildLaunchData(common_http_Request $request)
+    {
+        return LtiLaunchData::fromRequest(
+            $request,
+            [LtiLaunchData::LIS_OUTCOME_SERVICE_URL => _url('manageResults', 'ResultController', 'taoLtiConsumer')]
```

Comment:
Why do we add LIS_OUTCOME_SERVICE_URL parameter and not LIS_RESULT_SOURCEDID in this call?